apps/health_campaign_field_worker_app/lib/blocs/delivery_intervention/deliver_intervention.dart

```diff
@@ -105,37 +121,50 @@ class DeliverInterventionBloc
     }
   }
 
+  // Event handler for handling active cycle and dose selection
   FutureOr<void> _handleActiveCycleDose(
     DeliverInterventionActiveCycleDoseSelectionEvent event,
     BeneficiaryRegistrationEmitter emit,
   ) async {
+    // Calculate the current running cycle based on start and end dates
     final currentRunningCycle = (event.projectType.cycles?.firstWhere((e) =>
             (e.startDate ?? 1696032000000) <=
```

Comment:
Why we have hardcoded the time

Reply:
@Ramkrishna-egov has removed the time in his branch @naveen-egov